plugins/out_datadog/datadog_remap.h

```diff
@@ -22,10 +22,12 @@
 
 #include "datadog.h"
 
+typedef int (*remap_to_tag_fn)(const char*, msgpack_object, flb_sds_t*);
```

Comment:
Potential namespace pollution. Please rename to something like dd_attr_remap_to_tag_fn, if that makes sense. Following a the prefix convention should help mitigate the namespace collisions.